esbuild-node-externals/src/index.ts

```diff
@@ -40,6 +40,10 @@ export const nodeExternalsPlugin = (paramsOptions: Options = {}): Plugin => {
     setup(build) {
       // On every module resolved, we check if the module name should be an external
       build.onResolve({ namespace: 'file', filter: /.*/ }, (args) => {
+        if (options.allowList.includes(args.path)) {
```

Comment:
Thanks for the pr, can you just add a comment explaining what the condition is used for please?